utils/install-dependencies.sh

```diff
@@ -45,31 +45,25 @@ function install_dependencies_with_aur() {
 # Install dependencies on centos and fedora
 function install_dependencies_with_yum() {
     sudo yum install -y yum-utils
-
-    local common_dep="curl wget git gcc openresty-openssl111-devel unzip pcre pcre-devel openldap-devel"
-    if [ "${1}" == "centos" ]; then
-        # add APISIX source
-        local apisix_pkg=apache-apisix-repo-1.0-1.noarch
-        rpm -q --quiet ${apisix_pkg} || sudo yum install -y https://repos.apiseven.com/packages/centos/${apisix_pkg}.rpm
-
-        # install apisix-runtime and some compilation tools
-        # shellcheck disable=SC2086
-        sudo yum install -y apisix-runtime $common_dep
-    else
-        # add OpenResty source
-        sudo yum-config-manager --add-repo "https://openresty.org/package/${1}/openresty.repo"
-
-        # install OpenResty and some compilation tools
-        # shellcheck disable=SC2086
-        sudo yum install -y openresty $common_dep
+    sudo yum-config-manager --add-repo "https://openresty.org/package/${1}/openresty.repo"
+    if [[ "${1}" == "centos" ]]; then
+        sudo yum -y install centos-release-scl
+        sudo yum -y install devtoolset-9 patch wget
+        set +eu
+        source scl_source enable devtoolset-9
+        set -eu
     fi
+    sudo yum install -y  \
+        gcc gcc-c++ curl wget unzip xz gnupg perl-ExtUtils-Embed cpanminus patch \
+        perl perl-devel pcre pcre-devel openldap-devel \
+        openresty-zlib-devel openresty-pcre-devel
```

Comment:
What do these dependencies do? There was no such thing before

Reply:
Previously, apsix-runtime was not built from source code.